utils/validator.go

```diff
@@ -48,3 +49,13 @@ func extractJson(fld reflect.StructField) string {
 
 	return name
 }
+
+// ValidatePractitionerContactDetails checks if the telephone number and email are missing
+// in the request body. If they are missing, the method returns a human-readable error message.
+func ValidatePractitionerContactDetails(practitioner models.PractitionerRequest) string {
```

Comment:
I think this validation function would be better placed in service/insolvency_resource.go, as it is specific to the insolvency resource, rather than in a generic util function.